docusaurus/src/css/custom.css

```diff
@@ -103,3 +103,11 @@ html[data-theme="dark"] .navbar.navbar .navbar-sidebar .navbar__brand .navbar__l
         max-width: 38px;
     }
 }
+
+/* ==========================================================================
+   Sidebar tweaks
+   ========================================================================== */
+
+html[data-theme="dark"] .sidebarLogo_node_modules-\@docusaurus-theme-classic-src-theme-DocSidebar- img {
```

Comment:
This class name does exist in production builds.

Reply:
Will do a less worse hack for now and we can check with @satazor if we can have a proper logo for dark mode created

Reply:
See https://github.com/moxystudio/react-native-with-moxy/pull/47#discussion_r451826619